popcorn/__main__.py

```diff
@@ -1,9 +1,9 @@
 #!/usr/bin/env python
 from __future__ import division
```

Comment:
These from __future__ imports aren't needed in python 3 so you should remove them in this PR.

Reply:
Removed the unused import from the future library.